docs/data-collection/scanning.md

```diff
@@ -598,6 +598,8 @@ It is critical to check that physiological signals are looking good:
 - [ ] Check the box confirming that it has been acquired in the issue collecting notes about the session.
     Don't forget to report any observations there.
 
+- [ ] Copy data from the ./data subfolder in the GitHub repository HCPh-fMRI-tasks into the synchronized Dropbox folder, and make sure the Wi-Fi is on for synchronization.
```

Comment:
Did you have to do this manually in every session?

Reply:
Yes